src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add: (value) => {
+      return (obj.result += value);
+    },
+
+    subtract: (value) => {
+      return (obj.result -= value);
+    },
+
+    multiply: (value) => {
+      return (obj.result *= value);
+    },
+
+    divide: (value) => {
+      return (obj.result /= value);
```

Comment:
You need to add a check for 0, because you cannot divide by 0 according to the rules of mathematics